squidb/src/com/yahoo/squidb/sql/Function.java

```diff
@@ -129,10 +129,10 @@ protected String separator() {
     }
 
     /**
-     * Create a Function that counts all rows
+     * Create a Function that counts all rows (i.e. count(*))
      */
     public static Function<Integer> count() {
-        return new ArgumentFunction<Integer>("COUNT", 1);
+        return new RawFunction<Integer>("COUNT(*)");
```

Comment:
From the SQLite docs: "The count(X) function returns a count of the number of times that X is not NULL in a group. The count(*) function (with no arguments) returns the total number of rows in the group."

So, I think count(1) and count(*) effectively end up equivalent, but a raw function that is just the literal count(*) seemed cleaner to me than binding a 1.